README.md

```diff
@@ -434,7 +433,7 @@ If you'd like to customize this list, modify the example code in ```config/initi
 
 ```ruby
 config.exception_level_filters.merge!({
-  'ActiveRecord::RecordNotFound' => 'ignore',
+  'ActionController::RoutingError' => 'ignore',
```

Comment:
Ah, that may be true... but def not the only source of a 404. It kind of
depends on what you catch and/or coerce.